lib/ssh_scan/api.rb

```diff
@@ -113,17 +113,25 @@ class API < Sinatra::Base
       get '/scan/results' do
         uuid = params[:uuid]
 
-        return {"completed" => false}.to_json if uuid.empty?
+        return {"scan" => "not found"}.to_json if uuid.nil? || uuid.empty?
 
         settings.db.find_scan_result(uuid)
       end
 
-      post '/scan/results/delete/:uuid' do
-        uuid = params['uuid']
-
-        return {"deleted" => "false"}.to_json if uuid.empty?
+      post '/scan/results/delete' do
+        uuid = params[:uuid]
 
-        settings.db.delete_scan(uuid)
+        if uuid.nil? || uuid.empty?
+          return {"deleted" => "false"}.to_json
+        else
+          scan = settings.db.find_scan_result(uuid)
+          if scan.empty?
+            return {"deleted" => "false"}.to_json
+          else
+            settings.db.delete_scan(uuid)
+            return {"deleted" => "true"}.to_json
```

Comment:
@royalharsh I added this, because the true case was never reporting a true feedback.